db/fixtures/ae_datastore/ManageIQ/System/CommonMethods/Quota.class/__methods__/validate_quota.rb

```diff
@@ -0,0 +1,457 @@
+
+def request_info
+  type = $evm.root['quota_type']
+  if type == 'service'
+    @service = true
+    @miq_request = $evm.root['service_template_provision_task']
+  else
+    @miq_request = $evm.root['miq_request']
+  end
+  $evm.log(:info, "Request: #{@miq_request.description} id: #{@miq_request.id} ")
+end
+
+def entity_info
+  user = @miq_request.requester
+  $evm.log(:info, "Quota Entity: #{user.current_group.description}")

```

Comment:
@tinaafitz 
The current_group might keep changing, so we might need the group from the provision request.